api4/system.go

```diff
@@ -444,3 +452,51 @@ func pushNotificationAck(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	ReturnStatusOK(w)
 }
+
+func setServerBusy(c *Context, w http.ResponseWriter, r *http.Request) {
+	if !c.App.SessionHasPermissionTo(c.App.Session, model.PERMISSION_MANAGE_SYSTEM) {
+		c.SetPermissionError(model.PERMISSION_MANAGE_SYSTEM)
+		return
+	}
+
+	// number of seconds to keep server marked busy
+	secs := r.URL.Query().Get("seconds")
```

Comment:
Should we consider max limit value of "seconds" to prevent setting too high when API is accessed other than webapp?

Reply:
I added a maximum via const. Currently set to one day, but easily changed. Via https://github.com/mattermost/mattermost-server/pull/13212/commits/946d716ce21c3cdca1ddf7ab328ea7eb59819519

I also fixed the order for i18n.